tests/by-util/test_tail.rs

```diff
@@ -95,8 +95,8 @@ fn test_follow_stdin() {
         .stdout_is_fixture("follow_stdin.expected");
 }
 
-// FixME: test PASSES for usual windows builds, but fails for coverage testing builds (likely related to the specific RUSTFLAGS '-Zpanic_abort_tests -Cpanic=abort')
-#[cfg(not(windows))]
+// FixME: test PASSES for usual windows builds, but fails for coverage testing builds (likely related to the specific RUSTFLAGS '-Zpanic_abort_tests -Cpanic=abort')  This test also breaks tty settings under bash requiring a 'stty sane' or reset.
+#[cfg(disable_until_fixed)]
```

Comment:
I am confused, why this regressed ?
new version of the libc crate ?

Reply:
btw why disabling it everywhere ? Linux should still work ?

Reply:
Orthogonal issue see  https://github.com/uutils/coreutils/issues/1719 . 'cargo test' breaks the bash terminal when this runs. You have to either 'stty sane' or reset.

Reply:
If I recall it didn't break the terminal under zsh. I didn't try other shells.

Reply:
ok, I am using zsh, this is probably why I don't see it.

Could you please do a separate PR for this ?
I am not a big fan of mixing changes in the same PR